django/contrib/admin/static/admin/css/rtl.css

```diff
@@ -254,3 +258,7 @@ div.colM {
 .submit-row input {
     float: left;
 }
+
+.related-widget-wrapper {
```

Comment:
This appears in the section labeled "IE7 specific bug fixes" -- is it correct? I suppose we should remove that section since IE7 is no longer supported.